src/rgw/rgw_process.cc

```diff
@@ -461,7 +461,7 @@ int process_request(rgw::sal::Store* const store,
   if (latency) {
     *latency = lat;
   }
-  dout(1) << "====== req done req=" << hex << req << dec
```

Comment:
is this change applicable to rados backend as well ?

Reply:
Yes. For rados as backend also the request is not correctly printed in the logs. It prints the address instead of the req id.

Reply:
Tested these changes on plain ceph as well. It works fine with ceph.

```
2022-04-19T06:02:59.350-0600 7f3fbcdf0700  1 ====== starting new request req=16554218101738952371 =====
2022-04-19T06:02:59.350-0600 7f3fbcdf0700  2 req 16554218101738952371 0.000000000s initializing for trans_id = tx00000e5bc65a7d07dd6b3-00625ea4f3-5e68-default
2022-04-19T06:02:59.350-0600 7f3fbcdf0700 10 req 16554218101738952371 0.000000000s rgw api priority: s3=8 s3website=7
2022-04-19T06:02:59.351-0600 7f3fbcdf0700 10 req 16554218101738952371 0.000999997s host=localhost
2022-04-19T06:02:59.351-0600 7f3fbcdf0700 20 req 16554218101738952371 0.000999997s subdomain= domain= in_hosted_domain=0 in_hosted_domain_s3website=0
...
2022-04-19T06:02:59.357-0600 7f3fb15d9700  2 req 16554218101738952371 0.006999983s s3:list_buckets op status=0
2022-04-19T06:02:59.357-0600 7f3fb15d9700  2 req 16554218101738952371 0.006999983s s3:list_buckets http status=200
2022-04-19T06:02:59.357-0600 7f3fb15d9700  1 ====== req done req=16554218101738952371 op status=0 http_status=200 latency=0.006999983s ======
```

Reply:
@dang, @mattbenjamin, @cbodley, what do you guys think about this change?

Reply:
looks good to me